package.json

```diff
@@ -14,7 +14,9 @@
     "storybook": "storybook dev -p 6006 -c storybook",
     "storybook:preview": "NODE_ENV=production pnpm storybook --docs",
     "storybook:clean": "rm -rf storybook/public",
-    "storybook:build": "storybook build --test -c storybook -o storybook/public --docs",
+    "storybook:build": "storybook build -c storybook -o storybook/public",
+    "storybook:build:test": "pnpm storybook:build --test",
+    "storybook:build:prod": "pnpm storybook:build --docs",
```

Comment:
Was contemplating whether to use `:site` instead of `:prod`. Or maybe `:docs`. Can change to whatever makes the most sense.

Reply:
I think this is fine, its just an internal script being terse is probably better